### DISCUSSION TITLE
cpg0039-garcia-fossa-livecellpainting AND cpg0040-garcia-fossa-AgNP

### DISCUSSION THREAD

**Discussion by ErinWeisbart** [broadinstitute/cellpainting-gallery#97] [Dataset descriptions]
Documentation repo = https://fefossa.github.io/2023_CourseImageAnalysis_NanoCell/index.html

**External Source**
Institution identifier = unicamp
Imaging Platform internal "champion" = Erin
- [x] Champion has requested prefix be opened for transfer to `staging-cellpainting-gallery` by emailing Ank with prefix. (Use of credentials [documented here](https://github.com/broadinstitute/cytoskel/blob/aws_docs/cytoskel/docs/access_cpg_staging.md))

**Transfer data to `staging-cellpainting-gallery`**
For each data subtype, mark off as complete or explain why it is not being included:
- [x] Image acquisition complete and uploaded (`/images`)
- [x] Feature extraction complete and uploaded (`/workspace/analysis`)
- [x] Backend complete and uploaded (`/workspace/backend`)
- [x] Load_data complete and uploaded (`/workspace/load_data_csv`)
- [x] Profiling complete and uploaded (`/workspace/profiles`)
- [x] Metadata complete and uploaded (`/workspace/metadata`)

Project completion (dataset being made public):
- [ ] Run [validation script](http://github.com/broadinstitute/cpg/cpgdata) to ensure data is completely uploaded
- [ ] Update cellpainting-gallery/README.md to add dataset to README
- [x] Transfer from `staging-cellpainting-gallery` to `cellpainting-gallery`

Optional preparation for publication:
- [ ] Run Distributed-BioFormats2Raw to create .ome.zarr files
- [ ] Upload (meta)data to IDR (images remain hosted in cellpainting-gallery) 
- [ ] Make IDR entry public

Once a paper is published:
- [ ] Update [Publications](https://github.com/broadinstitute/cellpainting-gallery/docs/publications.md)
- [ ] Add the paper to https://github.com/cytodata/awesome-cytodata

**Comment by ErinWeisbart** [comment]
As initially described:
`cpg0039-garcia-fossa-livecellpainting`
The dataset includes results from Live Cell Painting assay, all folders in this dataset contain images captured in two channels: GFP and Propidium Iodide. These correspond to the LED cubes used for imaging on the Cytation 5 device: GFP (excitation/emission = 469/525 nm) and Propidium Iodide (excitation/emission = 531/647 nm)

`cpg0040-garcia-fossa-AgNP`
The dataset includes results from three distinct cell imaging experiments using fluorescence microscopy. In the first experiment, Live Cell Painting, images were captured in two channels: GFP (469/525 nm) for live cell staining and Propidium Iodide (531/647 nm) for detecting dead cells, using the Cytation 5 imaging system. In the second experiment, a modified version of the Cell Painting assay was used, capturing images with DAPI for nuclei, GFP for the endoplasmic reticulum, Propidium Iodide for actin, Golgi, and plasma membrane, and Cy5 for mitochondria. In the final experiment, the ToxPath panel, DAPI (Hoechst) was used for nuclei, GFP for neutral lipids to assess lipidosis, Texas Red for γH2AX as a marker for DNA damage, and Cy5 for NFκB to indicate inflammation.

Bispo, can you clarify the difference between the LCP data in cpg0039 and cpg0040? From my standpoint, it would make sense to put both LCP datasets into cpg0039 and have the two modified Cell Painting panels in cpg0040.

**Reply by labnanocell** [reply]
Erin, the Live Cell Painting (LCP) assay is the same in both datasets. However, the difference is that cpg0039 contains only LCP, while cpg0040 includes two additional panels: Cell Painting and ToxPath. The ToxPath panel is customized and includes the following stains: DAPI (Hoechst for nuclei staining), GFP (neutral lipid staining to assess lipidosis), Texas Red (γH2AX protein staining), and Cy5 (NFκB inflammation indicator staining). Fernanda and I have prepared a detailed description, which I’ve included below.

# Live Cell Painting Project Dataset (`cpg0039-garcia-fossa-livecellpainting`)

This dataset contains all the images used in the Live Cell Painting project, which aimed to create an image-based profiling assay for analyzing cell morphological changes induced by silver nanoparticles. The goal was to achieve this using a simple and cost-effective approach. After careful screening, **Acridine Orange** was chosen for its metachromatic properties and its ability to stain DNA, RNA, and acidic vesicles. Two channels were used to acquire fluorescence images with Acridine Orange.

All folders in this dataset contain images captured in two channels: **GFP** and **Propidium Iodide**. These correspond to the LED cubes used for imaging on the Cytation 5 device:
- **GFP** (excitation/emission = 469/525 nm)
- **Propidium Iodide** (excitation/emission = 531/647 nm)

The main directory for the project is named `2022_09_09_LiveCellPainting_fossa_Cimini`.

## Folder Organization

Within each subproject folder, there are two primary subfolders:
- `illum` – Contains the illumination correction function.
- `images` – Contains the raw images acquired.

For more details on the folder structure, refer to the project documentation. Within each `illum` or `images` folder, you will find subfolders named after the plate replicates. Inside each `images` folder, there is a `load_csv` folder containing CSV files used as inputs for analysis in **CellProfiler**.

## Image Usage for Specific Analyses

- **Figure 2**: Images used to demonstrate Acridine Orange capturing the dynamics of acidic vesicles are found in the folders `2022_03_24_Acidification_AO` (Acridine Orange images) and `2022_03_24_Acidification_LT` (Lysotracker images used in Supplemental Figure 2).
- **Figure 3**: Images related to silver nanoparticles (AgNP) analysis are located in the folder `2021_10_08_AgNPViability`.
- **Figure 4**: Images for DILI-induced profiling analysis are located in the folder `2022_06_07_DILI`.

The complete image analysis workflow for this dataset is described in detail in [this online book](#).

## Folder Structure

2022_09_09_LiveCellPainting_fossa_Cimini 
├── 2021_10_08_AgNPViability 
├── 2022_03_24_Acidification_AO
├── 2022_03_24_Acidification_LT 
└── 2022_06_07_DILI

--- 

# AgNP Exposure Project Dataset (`cpg0040-garcia-fossa-AgNP`)

This dataset contains all images from the AgNP (silver nanoparticles) exposure project, which aimed to analyze the image-based profile of **Huh-7 hepatocytes** exposed to silver nanoparticles under two conditions:
- **Acute exposure** (1 day, higher concentration of AgNP)
- **Repeated exposure** (15 days, lower concentration of AgNP)

Cells were stained using three imaging panels:
- **Live Cell Painting** (`2022_06_02_LiveCellPainting` folder)
- **Cell Painting** (`2022_06_05_CellPainting` folder)
- **Tox Painting** (`2022_06_07_ToxPath` folder)

Each experiment was performed in two independent replicates, and thus each folder contains subfolders for each replicate plate.

## Folder Organization

Within each subproject folder, you will find:
- `illum` folder: Contains the illumination correction function.
- `images` folder: Contains the raw acquired images.

For detailed folder organization, refer to the project documentation. Inside each `illum` or `images` folder, subfolders are named after plate replicates. Additionally, the `images` folder contains a `load_csv` folder with CSV files used as input for **CellProfiler** analysis.

## Imaging Panels and Channels

### 2022_06_02_LiveCellPainting

Images were acquired in two channels:
- **GFP** (excitation/emission = 469/525 nm)
- **Propidium Iodide** (excitation/emission = 531/647 nm)

These channels correspond to the LED cubes on the Cytation 5 imaging system. For more details on the Live Cell Painting assay, refer to the original paper.

### 2022_06_05_CellPainting

- **DAPI** (cytoplasm and nuclei staining)
- **GFP** (endoplasmic reticulum staining)
- **Propidium Iodide** (actin, Golgi, and plasma membrane staining)
- **Cy5** (mitochondria staining)

For more details on the Cell Painting assay, refer to the original paper.

### 2022_06_07_ToxPath

- **DAPI** (nuclei staining with Hoechst)
- **GFP** (neutral lipid staining to check for lipidosis)
- **Texas Red** (γH2AX protein staining)
- **Cy5** (NFκB inflammation indicator staining)

## Data Analysis

All images were analyzed using **CellProfiler** and **Python**. Detailed information on pre-processing and data analysis can be found in [this online book](#).

## Folder Structure
2023_01_04_AgNPContinuousAcute_fossa_Cimini 
├── 2022_06_02_LiveCellPainting 
├── 2022_06_05_CellPainting 
└── 2022_06_07_ToxPath

**Reply by ErinWeisbart** [reply]
Thanks for clarifying @labnanocell .
The data analysis links didn't format correctly - can you post them again here?

**Reply by fefossa** [reply]
Hi Erin! Sorry about the delay on this.
The link for the book is: https://fefossa.github.io/2023_CourseImageAnalysis_NanoCell/index.html

**Comment by ErinWeisbart** [comment]
I'm needing some clarification on mapping between artifacts from our imaging-platform-ssf S3 from when Fernanda was visiting our lab to what was written above by @labnanocell and uploaded to CPG staging as there is currently a mismatch.

The images and workspace that were uploaded to CPG-staging for `cpg0040-garcia-fossa-AgNP` are:
```
2022_05_25_LiveCellPainting
- 220526_084043_Plate_1
- 220601_155638_Plate_1
- 220609_090651_Plate_1

2022_05_28_CellPainting
- 220528_102915_Plate_1
- 220604_111529_Plate_1
- 220609_145227_Plate_1

2022_05_30_ToxPath
- 220530_150503_Plate_1
- 220610_164559_Plate_1
- 220613_154914_Plate_1
 ```
These map to our `2022_10_04_AgNPCellRecovery_fossa_Cimini`

The notes above from @labnanocell  say that `cpg0040-garcia-fossa-AgNP` should map to `2023_01_04_AgNPContinuousAcute_fossa_Cimini` which are very similarly named but different folders:
```
2022_06_02_LiveCellPainting
- 220602_145121_Plate_1
- 220616_143137_Plate_1

2022_06_05_CellPainting
- 220605_085428_Plate_1
- 220617_134428_Plate_1

2022_06_07_ToxPath
- 220607_154812_Plate_1
- 220620_083410_Plate_1
```

The description above also says: "Each experiment was performed in two independent replicates, and thus each folder contains subfolders for each replicate plate". This is consistent with what is in our `2023_01_04_AgNPContinuousAcute_fossa_Cimini` but not the 3 subfolders each from `2022_10_04_AgNPCellRecovery_fossa_Cimini`

@fefossa @labnanocell Can you please help clarify:
- Should `cpg0040-garcia-fossa-AgNP` be the described `2023_01_04_AgNPContinuousAcute_fossa_Cimini` instead of what was uploaded?
- If so, what should be done with the `2022_10_04_AgNPCellRecovery_fossa_Cimini` that are currently uploaded as `cpg0040-garcia-fossa-AgNP` - can we keep it in the gallery? can it stay in the same prefix? if so can you describe what it is and how it's different from `2022_10_04_AgNPCellRecovery_fossa_Cimini` ?

**Reply by ErinWeisbart** [reply]
After discussion with @labnanocell we have determined that:
`cpg0039-garcia-fossa-livecellpainting` contains `2022_09_09_LiveCellPainting_fossa_Cimini`
`cpg0040-garcia-fossa-AgNP` contains `2022_10_04_AgNPCellRecovery_fossa_Cimini` AND `2023_01_04_AgNPContinuousAcute_fossa_Cimini`

So `cpg0040-garcia-fossa-AgNP` contains all of:
```
2022_05_25_LiveCellPainting
- 220526_084043_Plate_1
- 220601_155638_Plate_1
- 220609_090651_Plate_1

2022_05_28_CellPainting
- 220528_102915_Plate_1
- 220604_111529_Plate_1
- 220609_145227_Plate_1

2022_05_30_ToxPath
- 220530_150503_Plate_1
- 220610_164559_Plate_1
- 220613_154914_Plate_1

2022_06_02_LiveCellPainting
- 220602_145121_Plate_1
- 220616_143137_Plate_1

2022_06_05_CellPainting
- 220605_085428_Plate_1
- 220617_134428_Plate_1

2022_06_07_ToxPath
- 220607_154812_Plate_1
- 220620_083410_Plate_1
```

**Reply by ErinWeisbart** [reply]
@labnanocell Can you please edit [your original dataset description](https://github.com/broadinstitute/cellpainting-gallery/discussions/97#discussioncomment-11157669) to include a description of the second set of plates in `cpg0040-garcia-fossa-AgNP` or add the additional information below?

**Reply by labnanocell** [reply]
@ErinWeisbart I've tried to update the description to make clear the experiments and folder structure. Can you check if it's better now?
 


# AgNP Exposure Project Dataset (cpg0040-garcia-fossa-AgNP)

This dataset contains raw microscopy images and illumination correction files from an **in vitro silver nanoparticle (AgNP) exposure study** in **Huh-7 human hepatocytes**. The goal is to quantify **phenotypic and toxicity-related responses** using multiplexed image-based assays across two exposure regimens.

## Study design and where to find each experiment

### 1) Acute exposure (1 day; higher AgNP concentration)
**Folder:** `2022_10_04_AgNPCellRecovery_fossa_Cimini/`  
This directory contains the full imaging output for the acute (single-day) exposure experiment, organized by imaging panel (subfolders listed below). Each panel was run in **two independent plate replicates**, stored as subfolders inside `images/` and `illum/`.

### 2) Repeated exposure (15 days; lower AgNP concentration)
**Folder:** `2023_01_04_AgNPContinuousAcute_fossa_Cimini/`  
This directory contains the full imaging output for the repeated (15-day) exposure experiment, with the same organization by imaging panel and **two independent plate replicates**.

## Imaging panels (subfolders inside each exposure regimen)

Within **each** regimen folder (acute or repeated), images are grouped by assay/panel:

- `2022_06_02_LiveCellPainting/`  
- `2022_06_05_CellPainting/`  
- `2022_06_07_ToxPath/` *(or `2022_05_30_ToxPath/` in the acute folder—see note below)*

Each panel folder contains:
- `images/` — raw acquired images  
- `illum/` — illumination correction functions for the same plates  
- `images/load_csv/` — CSV inputs used to run the CellProfiler pipelines (plate maps / load data)

**Replicates:** Inside `images/` and `illum/`, subfolders are named by **plate replicate** (e.g., replicate 1 and replicate 2). These correspond to independent experimental runs of the same condition set.

## Channels per imaging panel

### Live Cell Painting — `2022_06_02_LiveCellPainting`
Two-channel live readout:
- **Green channel** (e.g., 469/525 nm): live-cell compatible dye signal  
- **Red channel** (e.g., 531/647 nm): **propidium iodide** (cell death / membrane integrity)

### Cell Painting — `2022_06_05_CellPainting`
Four-channel multiplexed morphology panel:
- **DAPI/Hoechst**: nuclei  
- **Green channel**: endoplasmic reticulum  
- **Red channel**: actin / Golgi / plasma membrane  
- **Cy5**: mitochondria

### Tox Painting / ToxPath — `2022_06_07_ToxPath` (or `2022_05_30_ToxPath`)
Four-channel toxicity pathway panel:
- **DAPI/Hoechst**: nuclei  
- **Green channel**: neutral lipids (lipidosis proxy)  
- **Texas Red**: γH2AX (DNA damage proxy)  
- **Cy5**: NFκB (inflammation proxy)

## Analysis overview

Images were processed using **CellProfiler** pipelines (illumination correction + feature extraction) and downstream analysis in **Python**. The `load_csv/` files inside each panel provide the plate-level metadata required to reproduce image loading and pipeline execution.

## Folder structure (high-level)

### Acute exposure
```text
2022_10_04_AgNPCellRecovery_fossa_Cimini/
├── 2022_06_02_LiveCellPainting/
├── 2022_06_05_CellPainting/
├── 2022_05_30_ToxPath/   (or 2022_06_07_ToxPath)
└── workspace/
```

### Repeated exposure
```text
2023_01_04_AgNPContinuousAcute_fossa_Cimini/
├── 2022_06_02_LiveCellPainting/
├── 2022_06_05_CellPainting/
├── 2022_06_07_ToxPath/
└── workspace/
```

## Metadata

### Subject
Medicine, Health, and Life Sciences

### Keywords
High-content imaging; high-content analysis; phenotypic profiling; nanotoxicology; silver nanoparticles; hepatocytes

**Comment by ErinWeisbart** [comment]
**Dataset Completion**:

`cpg0039-garcia-fossa-livecellpainting` is missing `illum` for batches `2022_03_24_Acidification_AO` and `2022_03_24_Acidification_LT`.

**Comment by ErinWeisbart** [comment]
@labnanocell Can you help clarify some metadata for me on `cpg0040-garcia-fossa-AgNP`

`Live Cell Painting`
- These only have acridine orange added to the samples as a label but the acridine orange is read out in two separate channels (Green/GFP and Red/PI). Propidium iodide has not been added to the samples, correct?

`Cell Painting`
- For each of the channels you imaged for Cell Painting (DNA/DAPI, ER/Green, AGP/Red, Mito/Cy5):
  - what are the excitation peak and emission peak of the channels on your microscope?
  - what are the actual dyes you use? (i.e. the name of the kit you used if it's a commercial kit or if you used individual reagents than the name of the dye and fluorophore - e.g. Hoechst 33342, MitoTracker Deep Red)

`Tox Painting`
- For each of the channels you imaged for Tox Painting (Hoechst/DAPI, neutral lipids/Green, γH2AX/Red, NFκB/Cy5):
  - what are the excitation peak and emission peak of the channels on your microscope?
  - what are the actual dyes you use? (i.e. the name of the kit you used if it's a commercial kit or if you used individual reagents than the name of the dye/antibody and fluorophore - e.g. Hoechst 33342, mouse anti–γH2AX IgG and Alexa Fluor 488 donkey anti-mouse IgG)
